src/main/java/Duke.java

```diff
@@ -1,10 +1,142 @@
+import java.util.Scanner;
+
+
 public class Duke {
+
+    private static Task[] tasks = new Task[100];
+    private static int taskCounter =0;
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
+
+        startMsg();
+        Scanner input = new Scanner(System.in);
+
+        while (input.hasNextLine()) {
+
+            String command = input.nextLine().trim();
+            String[] doneTokens = command.split(" ");
```

Comment:
i think the naming of this variable is a bit confusing. It is used to check mainly the first 2 word , and not just for "done". Also since it only checks the first two word i think do not really need to create an array of the whole sentence.